rosbag2_cpp/src/rosbag2_cpp/writers/sequential_writer.cpp

```diff
@@ -43,6 +43,12 @@ std::string format_storage_uri(const std::string & base_folder, uint64_t storage
 
   return (rcpputils::fs::path(base_folder) / storage_file_name.str()).string();
 }
+
+std::string strip_parent_path(const std::string & relative_path)
```

Comment:
Can this be `const`?

Reply:
This isn't a method

Reply:
I guess what I meant is:
```
const std::string strip_parent_path(...) { ... }
```
It looks like a function to me so not sure what you mean by this isn't a method.

Reply:
A method is a member function of a class - this is just a plain function. I thought you meant

```
std::string strip_parent_path(...) const {...}
```

What would be the point of returning a const string? It's returning by value so the user can declare the result const if they don't want anything changing it.

Reply:
why would you want to return a `const std::string` here?

Reply:
That answers my question, I was wondering if it was necessary or not.
Thanks @emersonknapp